src/libsync/vfs/cfapi/hydrationjob.h

```diff
@@ -71,6 +73,8 @@ class OWNCLOUDSYNC_EXPORT HydrationJob : public QObject
     void onGetFinished();
     void onGetCanceled();
 
+    bool _isCanceled = false;
```

Comment:
Please add it at the end to ease the life of the compiler if it should add padding to align the attributes in memory.